CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
 # Change Log
+
```

Comment:
were all these newlines and formatting changes in the README meant to be committed?

Reply:
no I didn't make those changes, not sure where they came from to be honest. I only added a CHANGELOG entry, I assume you are referring to the CHANGELOG and not README though right?

Reply:
yes I was :) - fixed it